src/test/java/org/prebid/server/bidder/rubicon/RubiconBidderTest.java

```diff
@@ -216,11 +217,11 @@ public void makeHttpRequestsShouldFilterAllImpressionsAndReturnErrorMeessagesWit
 
         // then
         assertThat(result.getErrors()).hasSize(3)
```

Comment:
Please, remove call to `hasSize()` and replace `containsExactly` with `containsExactlyInAnyOrder`